src/java.base/share/classes/sun/security/x509/X509Key.java

```diff
@@ -140,6 +140,27 @@ public static PublicKey parse(DerValue in) throws IOException
         return subjectKey;
     }
 
+    /*
+     * Parses X.509 subject public key DER and return it as a
+     * X509Key.  Useful for PKCS8v2.
+     */
+    public static X509Key parse(byte[] encoded) throws IOException
```

Comment:
Isn't this the same as `X509Key.parse(byte[])`?

Reply:
I'm assuming you mean `X509Key.decode()`?   It looks like that can be used instead of this method.